### PR TITLE
Fix crash when FK references unknown 'app_label'

### PR DESCRIPTION
**Pull request by flaeppe**
# I have made things!

Plugin previously crashed running the provided test case, now it yields an intended error message instead.

## Related issues

Ref: #1312

### CODE REVIEW

**Review by sobolevn — APPROVED** [review]
LGTM

File: tests/typecheck/fields/test_related.yml [hunk 1]
```diff
@@ -928,3 +928,18 @@
...
+        from installed.models import InstalledModel
+    installed_apps:
+      - installed
+    files:
+        -   path: installed/__init__.py
+        -   path: installed/models.py
+            content: |
+                from django.db import models
+                class InstalledModel(models.Model):
+                    non_installed = models.ForeignKey(  # E: Cannot find model 'not_installed.NonInstalledModel' referenced in field 'non_installed'
```
Comment (intgr) [inline tests/typecheck/fields/test_related.yml:943]: Is it also possible to ignore this error using a `# type: ignore` comment?

**Review by intgr — APPROVED** [review]
Cool, this makes sense.

File: tests/typecheck/fields/test_related.yml (hunk 1 above)
Comment (intgr) [reply tests/typecheck/fields/test_related.yml:943]: I tested it and indeed it works.

Comment (flaeppe) [reply tests/typecheck/fields/test_related.yml:943]: I didn't think mypy supported suppressing any errors produced by a plugin (yet?)

Refs:
- https://github.com/typeddjango/django-stubs/issues/1023#issuecomment-1178853194
- https://github.com/python/mypy/issues/12987

Comment (intgr) [reply tests/typecheck/fields/test_related.yml:943]: I thought the same. Yet the test passes successfully with the following diff:

```patch
diff --git a/tests/typecheck/fields/test_related.yml b/tests/typecheck/fields/test_related.yml
index 7e9758ae..cc863373 100644
--- a/tests/typecheck/fields/test_related.yml
+++ b/tests/typecheck/fields/test_related.yml
@@ -940,6 +940,6 @@
             content: |
                 from django.db import models
                 class InstalledModel(models.Model):
-                    non_installed = models.ForeignKey(  # E: Cannot find model 'not_installed.NonInstalledModel' referenced in field 'non_installed'
+                    non_installed = models.ForeignKey(  # type: ignore
                         "not_installed.NonInstalledModel", on_delete=models.CASCADE
                     )
```

I'm not complaining 😆

Comment (flaeppe) [reply tests/typecheck/fields/test_related.yml:943]: How interesting, have to compare how this error is emitted versus the one that couldn't be ignored

**Pull request merged**